aws-predictions/src/main/java/com/amplifyframework/predictions/aws/service/AWSTextractService.java

```diff
@@ -0,0 +1,232 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amplifyframework.predictions.aws.service;
+
+import android.graphics.RectF;
+import androidx.annotation.NonNull;
+
+import com.amplifyframework.core.Consumer;
+import com.amplifyframework.predictions.PredictionsException;
+import com.amplifyframework.predictions.aws.AWSPredictionsPluginConfiguration;
+import com.amplifyframework.predictions.aws.adapter.IdentifyTextResultTransformers;
+import com.amplifyframework.predictions.models.BoundedKeyValue;
+import com.amplifyframework.predictions.models.IdentifiedText;
+import com.amplifyframework.predictions.models.Polygon;
+import com.amplifyframework.predictions.models.Selection;
+import com.amplifyframework.predictions.models.Table;
+import com.amplifyframework.predictions.models.TextFormatType;
+import com.amplifyframework.predictions.result.IdentifyDocumentTextResult;
+import com.amplifyframework.predictions.result.IdentifyResult;
+import com.amplifyframework.util.UserAgent;
+
+import com.amazonaws.AmazonClientException;
+import com.amazonaws.ClientConfiguration;
+import com.amazonaws.auth.AWSCredentialsProvider;
+import com.amazonaws.mobile.client.AWSMobileClient;
+import com.amazonaws.services.textract.AmazonTextractClient;
+import com.amazonaws.services.textract.model.AnalyzeDocumentRequest;
+import com.amazonaws.services.textract.model.AnalyzeDocumentResult;
+import com.amazonaws.services.textract.model.Block;
+import com.amazonaws.services.textract.model.BlockType;
+import com.amazonaws.services.textract.model.DetectDocumentTextRequest;
+import com.amazonaws.services.textract.model.DetectDocumentTextResult;
+import com.amazonaws.services.textract.model.Document;
+import com.amazonaws.services.textract.model.FeatureType;
+import com.amazonaws.services.textract.model.Geometry;
+import com.amazonaws.services.textract.model.SelectionStatus;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Predictions service for performing text translation.
+ */
+final class AWSTextractService {
+    private final AmazonTextractClient textract;
+    private final AWSPredictionsPluginConfiguration pluginConfiguration;
```

Comment:
Looks like this is [dead-store](https://en.wikipedia.org/wiki/Dead_store)?

Reply:
It is indeed (for the time being)! I will add these in when I go over configuration for predictions after being finished, but both labels detection and text detection are missing configuration.

For now, I'm only preemptively adding the ones that are required for the plugin to work.